### ISSUE TITLE
Suggested readme update

### ISSUE THREAD

**Issue by larsbrinkhoff** [fsword7/ts10#1]
Hello,

Thank you for publishing this.  Although there was a copy on sourceforge, I think GitHub is a much better place for software today.

I'd like to suggest adding a few lines to the top-level ReadMe file:

- How to build this.  Which tools to use for building; which libraries and other depencencies are needed.
- How to run it.  Configurable options, target file/tape/disk image formats.
- Which target operating systems are supported.

This way, users can more quickly get up and running ts10.

**Comment by larsbrinkhoff** [comment]
I see some of this is well covered by the subdirectory ReadMe files.

**Comment by fsword7** [comment]
Yes, I will work on ReadMe file soon.  I was working on dump10 utility tool to now working on Windows.

I checked other ReadMe files in subfolders and found some useful information.  I will work on it to merge them into big document.
